xayn-ai/src/coi/system.rs

```diff
@@ -289,10 +293,10 @@ pub(crate) fn update_user_interests(
 fn log_document_view_time(
     cois: &mut Vec<PositiveCoi>,
     embedding: &Embedding,
-    config: &Configuration,
+    _config: &Configuration,
```

Comment:
if we don't need the config anymore here, we could just remove the argument.

Reply:
I wasn't sure whether we'd introduce other configuration that will be relevant here, and since it's a cascading change I just left it in for now. Do you think this is a must-have for getting this merged?

Reply:
not a must have, we can change it later on if needed as well.